Source/WebGPU/WGSL/tests/invalid/overload.wgsl

```diff
@@ -33,9 +35,9 @@ fn testConstraints() {
 }
 
 fn testBottomOverload() {
-  // CHECK-L: no matching overload for initializer vec2<f32>(<AbstractInt>)
-  let m = vec2<f32>(0);
```

Comment:
This was never correct, it failed at some point, but shouldn't have since it's valid to promote `AbstractInt`s to `f32`. Either way, the main purpose of this is to test that we don't trigger an extra error below, so I just made it fail again